### PR TITLE
Improve document selector output styling

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Minor document selector output styling improvements.

After:

Screenshot 2025-07-22 at 15 36 59

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: packages/client/src/v2-events/components/forms/inputs/FileInput/SingleDocumentPreview.tsx [hunk 1]
```diff
@@ -68,7 +65,7 @@ export function SingleDocumentPreview({
     return matchingOptionForDocType && matchingOptionForDocType.label
   }
   return (
-    <Wrapper id={`preview-list-${id}`}>
+    <Wrapper id={`preview-list-${id}`} {...props}>
```
Comment (tareq89) [inline packages/client/src/v2-events/components/forms/inputs/FileInput/SingleDocumentPreview.tsx:68]: What is the use of this `props` here?

Comment (cibelius) [reply packages/client/src/v2-events/components/forms/inputs/FileInput/SingleDocumentPreview.tsx:68]: This is so that the `className` prop created by styled components is passed and used on the Wrapper. This is needed to style an existing component: https://styled-components.com/docs/basics#styling-any-component
